src/Card/CardCarousel/CardCarousel.jsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { OverflowScroll } from '../..';
+import { CARD_DECK_ITEM_CLASS_NAME } from '../CardDeck';
+import CardCarouselProvider from './CardCarouselProvider';
+import CardCarouselHeader from './CardCarouselHeader';
+import CardCarouselItems from './CardCarouselItems';
+
+function CardCarousel({
+  children,
+  title,
+  subtitle,
+  columnSizes,
+  hasInteractiveChildren,
+  canScrollHorizontal,
```

Comment:
Just some initial testing locally. I noticed a small ui issue when horizontal scroll is enabled, but the number of elements in the OverflowScroll is does not require horizontal scrolling. The container for the horizontal scroll is remaining despite not being able to scroll through the card carousel

![Screen Shot 2023-01-03 at 9 50 49 AM](https://user-images.githubusercontent.com/82611798/210385346-cbdf02d5-9e66-4c26-85c0-6ffd9839c026.png)

Reply:
Going to defer on this for now.